src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -191,10 +194,12 @@ class RunWorkflowTests : SeleniumTest()
         createButton.click()
         wait.until(ExpectedConditions.presenceOfElementLocated(By.id("git-branch")))
 
-        assertThat(driver.findElements(By.cssSelector("#git-commit option")).count()).isEqualTo(1)
+        // We generally expect one git commit option in demo orderly before refresh, but have found that this can be two commits on Buildkite run. This may be related to having run a workflow in another test
+        assertThat(driver.findElements(By.cssSelector("#git-commit option")).count()).isIn(listOf(1, 2))
 
         val refreshButton = driver.findElement(By.id("git-refresh-btn"))
         refreshButton.click()
+        assertThat(driver.findElements(By.cssSelector("error-info")).count()).isEqualTo(0)
```

Comment:
Would `error-info` show in the DOM if there was an error or do I need to use some other selector to check for errors?

Reply:
Suggest modifying `api.get("/workflows")` to `api.get("/blah")` so that you get an error, then use Chrome web inspector to see what actually gets rendered for the `error-info` component (it won't be `<error-info>` because that's not a valid HTML element)